scripting/cwx/item_config.sp

```diff
@@ -305,10 +305,53 @@ int EquipCustomItem(int client, const CustomItemDefinition item) {
 			SetEntProp(itemEntity, Prop_Send, "m_iAccountID", accountid);
 		}
 	}
-	
-	// TODO: implement a version that nullifies runtime attributes to their defaults
-	SetEntProp(itemEntity, Prop_Send, "m_bOnlyIterateItemViewAttributes",
-			!item.bKeepStaticAttributes);
+
+	if(!item.bKeepStaticAttributes) {
+		TF2Attrib_RemoveAll(itemEntity);
+
+		char valueType[2];
+		char valueFormat[64];
+
+		int staticAttribs[16];
+		float staticAttribsValues[16];
+
+		TF2Attrib_GetStaticAttribs(item.defindex, staticAttribs, staticAttribsValues);
+
+		for(int i = 0; i < 16; i++)
+		{
+			if(staticAttribs[i] == 0) {
+				continue;
+			}
+
+			// Probably overkill
+			if(staticAttribs[i] == 796 || staticAttribs[i] == 724 || staticAttribs[i] == 817 || staticAttribs[i] == 834 
+				|| staticAttribs[i] == 745 || staticAttribs[i] == 731 || staticAttribs[i] == 746) {
+				continue;
+			}
+
+			// "stored_as_integer" is absent from the attribute schema if its type is "string".
+			// TF2ED_GetAttributeDefinitionString returns false if it can't find the given string.
+			if(!TF2Econ_GetAttributeDefinitionString(staticAttribs[i], "stored_as_integer", valueType, sizeof(valueType))) {
+				continue;
+			}
+
+			TF2Econ_GetAttributeDefinitionString(staticAttribs[i], "description_format", valueFormat, sizeof(valueFormat));
+
+			// Since we already know what we're working with and what we're looking for, we can manually handpick
+			// the most significative chars to check if they match. Eons faster than doing StrEqual or StrContains.
+
+			if(valueFormat[9] == 'a' && valueFormat[10] == 'd') { // value_is_additive & value_is_additive_percentage
```

Comment:
Fairly sure this also breaks Flame Throwers &mdash; one of the frequent complaints (with CW3) is that after Jungle Inferno, not preserving attributes disabled the new FT params.  Granted they'd still be broken with the existing `keep_static_attrs` implementation.

It doesn't show up in my attribute listing page but they're all `value_is_additive` in `items_game`.

Reply:
(Previously thought about in #2.)

Reply:
Fair, I thought that it could break some niche weapons that had attributes that affected the functionality of the weapon. In my personal case, I didn't pay too much attention to it as it'd just be a matter of setting the attribute back to what it has to be in the config file of the weapon. 
That's tricky. Would there even be a proper way to do it without hardcoding the attributes?

Reply:
With FT specifically, one could do a lookup to check the static attributes on the base item.  The other base items (excluding spellbook / grappling hook / MvM canteen) only have `weapon_stattrak_module_scale` and `kill eater score type *` attributes.  The FT does have `extinguish restores health` by default as well, though.

Reply:
There is not a proper way to retrieve the base item unfortunately ( without doing some ugly boilerplate, at least ). So honestly I'd just hardcode the attributes to skip since there is only a small amount anyway.